docs/src/main/paradox/s3.md

```diff
@@ -98,3 +98,26 @@ Java
     sbt
     > s3/test
     ```
+
+# Bluemix Cloud Object Storage with S3 API
```

Comment:
I wonder if we should also link to in the main connector listing, but just point it to this section of the S3 docs, to increase discoverability that it is supported?

Reply:
Yea, I'll fix that

Reply:
It looks like there is no easy fix for this, I'll merge this now and revisit later